pom.xml

```diff
@@ -57,6 +57,14 @@
         <role>developer</role>
       </roles>
     </developer>
+    <developer>
+      <id>rascheman</id>
```

Comment:
- we use ids as {first}{last}
- please include organizationUrl
- also update CONTRIBUTORS.md